lib/evaluate.js

```diff
@@ -152,7 +154,11 @@ export const evaluate = async ({
   try {
     recordTelemetry('retrieval_stats_honest', (point) => {
       point.intField('round_index', roundIndex)
-      buildRetrievalStats(honestMeasurements, point)
+      // DISCUSSION POINT: should we include minority measurements in these stats?
+      // Right now, we include majority measurements only. As a result,
+      // the retrieval-result-breakdown does not show outlier result types
+      // like infrequently-encountered HTTP error codes.
+      buildRetrievalStats(measurementsToReward, point)
```

Comment:
TODO: resolve this discussion.

> Should we include minority measurements in these stats?
> Right now, we include majority measurements only. As a result,
> the retrieval-result-breakdown does not show outlier result types
> like infrequently-encountered HTTP error codes.

I am leaning towards including minority results in "retrieval_stats_honest", but implementing such change in a new pull request.

Reply:
see 1392633 and https://github.com/filecoin-station/spark-evaluate/issues/446